### DISCUSSION TITLE
Connecting Github Org to Private LFS?

### DISCUSSION THREAD

**Discussion by Srixun** [community/community#114324] [New to GitHub]
Hey guys, Just curious if theres a way I can tie in our GitHub organization into the Git LFS On my SFTP server/file server?

We're a small private project for fun so just trying not to throw money at it unless necessary. :)

**Comment by queenofcorgis** [comment]
Hi @Srixun :octocat: , 

Would you mind sharing a bit more on what you mean by tie in and what you're trying to accomplish? 

Thanks!

**Reply by Srixun** [reply]
Hey @queenofcorgis  Thank you for your assistance. Ill try to explain better.

I have install Git LFS on a large fileserver I own.  intending to use it for a Github organization/Repo.

I believe I have everything linked properly. but my contributors are still unable to upload large files. 

Im hoping/thinking theres a way to just directly tie it so when members upload large files to the repo, it just moves them to my Git LFS for storage. Instead of having to purchase storage, if that makes sense?

Is what im thinking possible? If not, Darn, if so. What am I missing?

Thank you :)

**Reply by queenofcorgis** [reply]
Thanks for those details @Srixun and taking the time for the explanation ❤️ . 

Have you seen [this Gist](https://github.com/git-lfs/git-lfs/wiki/Tutorial#lfs-url)? It looks like what you're trying to accomplish may be possible.
